<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentOutbound</class>
 <widget class="QDialog" name="EquipmentOutbound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
background:rgba(85,170,225,0);
	font-style: italic bold;
	font size:14px;</string>
   </property>
   <property name="text">
    <string>装备出库管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>13</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_XML">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>100</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>XML</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_EXCEL">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>EXCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>装备编号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_types">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>装备类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_return">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>是否归还</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_returnVaild">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>出库天数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>出库日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_returntime">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>应归还日期</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_look">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_reason">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>原因</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
